.github/workflows/master.yaml

```diff
@@ -1,4 +1,4 @@
-name: Branch Build
+name: Master Build
```

Comment:
We have a main branch, please fix the naming here and in the filename

Reply:
done